### DISCUSSION TITLE
Availability of UR50/S, UR50/D also UR100/S and UR100/D

### DISCUSSION THREAD

**Discussion by j-fife** [facebookresearch/esm#667] [Q&A]
I was going through the methods of _Rives et al. PNAS (2021)_ I was looking for how to recreate the UR50/S and UR50/D splits of the original UniRef50 dataset but was not able to quite piece together how to do so. Are these datasets available anywhere for download (also UR100/S and UR100/D) or is there an available method of recreating them somewhere in the code base?

**Comment by ebetica** [comment]
I think I actually answered this over email but will put it here for posterity:

Here's the link: https://ftp.uniprot.org/pub/databases/uniprot/previous_releases/release-2021_04/uniref/

UR50/S is UR50 as downloaded from that site. UR50/D samples each cluster of UR50/S and then samples each sequence within the cluster at each training iteration, and can be created form the cluster members file in that download. Though we may have used linclust to generate it.
